selenium_webdriver/src/main/java/pageobject/sauceDemoPages/SauceDemoMainPage.java

```diff
@@ -0,0 +1,56 @@
+package pageobject.sauceDemoPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pageobject.basePage.BasePage;
+
+import java.util.List;
+
+public class SauceDemoMainPage extends BasePage {
+
+    @FindBy(id = "add-to-cart-sauce-labs-backpack")
+    private WebElement sauceLabsBackPackItemAddToCartButton;
+
+    @FindBy(id = "remove-sauce-labs-backpack")
+    private WebElement sauceLabsBackPackItemRemoveFromCartButton;
+
+    @FindBy(xpath = "//span[@class='shopping_cart_badge']")
+    public WebElement sauceLabsBackPackItemIsAddedToCart;
+
+    @FindBy(xpath = "//*[text()='Epic sadface: Sorry, this user has been locked out.']")
```

Comment:
you find the button by specific test and later in Test getLoginButtonErrorMessage() validate it). You need to use the common button locator e.g. //h3[@data-test='error']/button and validate the text

Reply:
Fixed